wolfssl/wolfcrypt/settings.h

```diff
@@ -3581,6 +3581,11 @@ extern void uITRON4_free(void *p) ;
     #define WOLFSSL_COPY_CERT
 #endif
 
+#if defined(OPENSSL_ALL) && !defined(WOLFSSL_NO_COPY_KEY)
+    #undef WOLFSSL_COPY_KEY
```

Comment:
Please add a code comment for purpose of `WOLFSSL_COPY_KEY` and use-case it resolves.

Reply:
Comments have been added.